backend/src/test/java/gov/cdc/usds/simplereport/db/repository/TestEventRepositoryTest.java

```diff
@@ -45,7 +46,7 @@ public void testLatestTestEventForPerson() {
         flush();
         TestEvent found = _repo.findFirst1ByPatientOrderByCreatedAtDesc(patient);
         assertEquals(second.getResult(), TestResult.UNDETERMINED);
-        List<TestEvent> found2 = _repo.findAllByCreatedAtInstant("2020-01-01T16:13:15.448000Z");
-        assertEquals(2, found2.size());
+        List<TestEvent> foundTestReports2 = _repo.findAllByCreatedAtInstant("2020-01-01T16:13:15.448000Z");
+        assertEquals(2, foundTestReports2.size() - foundTestReports1.size());
```

Comment:
This is very clever, but your intuition that there is a better way is correct. 😁